doc-content/enterprise-only/installation/nfs-server-configure-proc.adoc

```diff
@@ -1,29 +1,32 @@
 [id='nfs-server-configure-proc_{context}']
 
-= Configuring an NFS server
+= Configuring an NFS version 4 server
 
-A shared file system is required for a {CENTRAL} clustered environment and each client node must have access to the shared file system.
-
-You must deploy and configure an NFS version 4 server.
+An NFS version 4 server with a shared file system is required for a {CENTRAL} clustered environment and each client node must have access to the shared file system.
 
 .Procedure
 
 . Configure a server to export NFS version 4 shares. For instructions about exporting NFS shares on Red Hat Enterprise Linux, see _Exporting NFS shares_ in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_file_systems/index[_Managing file systems_]. For more information about creating the NFS server, see https://access.redhat.com/solutions/1355233[_How to configure NFS in RHEL 7_].
 
-. On the server, create an `/opt/kie/data` share with the `rw,sync,no_root_squash` options by adding the following line to the `/etc/exports` file:
+. On the server, open the `/etc/exports` file in a text editor.
+
+. Add the following line to the `/etc/exports` file where `<HOST_LIST>` is a space-separated list of IP addresses and options of hosts that are allowed connect to the server:
```

Comment:
[Typo] Add "to" or rephrase:

s/that are allowed connect to the server:/that are allowed to connect to the server:/that are authorized to connect to the server